deployment/terraform/assets/cluster.tf

```diff
@@ -37,16 +37,8 @@ locals {
   private_ip = data.external.private_ip.result.ip
 }
 
-data "aws_subnets" "selected" {
-  filter {
-    name   = "vpc-id"
-    values = [var.cluster_vpc_id]
-  }
-
-  filter {
-    name   = "availability-zone"
-    values = [var.aws_az]
-  }
+data "aws_vpc" "default" {
```

Comment:
Where are you using this?

Reply:
Good catch! It was used in the definition of `aws_subnets.selected`